cmd/bosun/conf/system_elastic_all.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 
 	"bosun.org/cmd/bosun/expr"
+	"github.com/bosun-monitor/bosun/slog"
```

Comment:
import here needs to be bosun.org/bosun/slog

Reply:
Typo? You mean to write "bosun.org/slog", right?

Reply:
oops yes